runtime/Cargo.toml

```diff
@@ -14,7 +14,7 @@ blake3 = "0.3.6"
 bv = { version = "0.11.1", features = ["serde"] }
 byteorder = "1.3.4"
 bzip2 = "0.3.3"
-dashmap = "3.11.10"
+dashmap = { version = "4.0.1", features = ["rayon"] }
```

Comment:
nits: there is "4.0.2". Also it seems that this major version bump isn't internal implementation overhaul.